docs/concepts/storage/volumes.md

```diff
@@ -481,6 +486,10 @@ spec:
 **Note:** The local PersistentVolume cleanup and deletion requires manual intervention without the external provisioner.
 {: .note}
 
+Starting in 1.9, local volume binding can be delayed by creating a StorageClass
```

Comment:
"can be delayed [until scheduling time]" ?

Reply:
done